theme/lumo/vaadin-combo-box-dropdown-styles.html

```diff
@@ -13,6 +13,7 @@
       }
 
       :host {
+        width: var(--vaadin-combo-box-overlay-width, var(--_vaadin-combo-box-overlay-default-width, auto));
```

Comment:
Should this be in `src` rather than copying the same code to both themes?

Reply:
Yeah, good catch, since this is not Lumo or Material specific. I thought about it at some point, but wanted to avoid creating a new style module just for this.

I'm not sure if it can somehow cleanly be added directly to the custom element definition here https://github.com/vaadin/vaadin-combo-box/blob/3921cbf84f504128a7d3408288c753eef3ba5e91/src/vaadin-combo-box-dropdown.html#L50 (since it's just inheriting the vaadin-overlay template now).
I think it's probably better/cleaner to just make a small style module for this right above the class? I think I'll do that unless you have a better suggestion.

Reply:
Updated the code. PTAL.